sklearn/naive_bayes.py

```diff
@@ -115,6 +115,16 @@ class GaussianNB(BaseNB):
 
     Read more in the :ref:`User Guide <gaussian_naive_bayes>`.
 
+    Parameters
+    ----------
+    fit_prior : boolean
+        Whether to learn class prior probabilities or not.
+        If false, a uniform prior will be used.
+
+    class_prior : array-like, size=(n_classes,)
+        Prior probabilities of the classes. If specified the priors are not
+        adjusted according to the data.
```

Comment:
Sorry, I didn't got it at first. You want to remove fit_prior and have something similar to the parameter max_feature of RandomForestClassifier for class_prior.

I can work on that. Shall I take care also about the DiscreteNB or it will be through another dedicated PR.